Extensions/magiccards_popups.js

```diff
@@ -0,0 +1,101 @@
+//* TITLE magiccards.info Popups **//
+//* VERSION 0.1.2 **//
+//* DESCRIPTION    Creates visual popups for links to cards on magiccards.info **//
+//* DEVELOPER thebombzen **//
+//* FRAME false **//
+//* BETA true **//
+
+/* globals XKit */
+
+
+/*
+ * magiccards_popups is an extension to New XKit. It creates popup images for links to http://magiccards.info/
+ * Note that it only creates popups for links to direct cards, not searches.
+ * It adds popups on your dashboard and official tumblr pages, but not on user blogs.
+ */
+
+XKit.extensions.magiccards_popups = new Object({
+
+    addPopup: function($atag, image_source){
+        var $img = $("<img>");
+        $img.prop("src", image_source);
+        $img.css({
+            "padding": "0px",
+            "border":  "0px",
+            "width": "312px",
+            "height": "445px",
+            "position": "fixed",
+            "visibility": "hidden",
+            "z-index": "300"
+        });
+        $atag.addClass("magiccards_popups-tooltip");
+        $atag.hover(function(){
+            $(this).children("img").last().css("visibility", "visible");
+        }, function(){
+            $(this).children("img").last().css("visibility", "hidden");
+        });
+        $atag.append($img);
+        $atag.on("mousemove.magiccards_popups", function(event){
+            var x = event.clientX, y = event.clientY;
+            if (y > window.innerHeight / 2){
+                y -= 485;
+            }
+            $(this).children("img").last().css({
+                "top": (y + 20) + 'px',
+                "left": (x + 20) + 'px'
+            });
+        });
+    },
+
+    addPopupsIfNecessary: function(){
+        if (XKit.interface.is_tumblr_page() !== true){
+            return;
+        }
+        $("a").each(function(){
+            var $atag = $(this);
+            if ($atag.hasClass("magiccards_popups-tooltip")){
+                return;
+            }
+            var old_href = $atag.prop("href"), theMatch, image_source, replaced;
+            if (old_href){
+                if (old_href.match(/^.*?\/\/magiccards.info\/.*?\/.*?\/.*?\.html$/)){
+                    theMatch = /^.*?\/\/magiccards.info\/(.*?)\/(.*?)\/(.*?)\.html$/.exec(old_href);
```

Comment:
That got me too on the first write-through. I don't know why they do that, but they do that.